src/components/random/Random.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import './Random.css';
+import contactsData from "../../contacts.json";
+
+
+
+const Random = ({setContacts, contacts}) =>{
+    
+    const addContact = () =>{
+        const select =  contactsData[Math.floor(Math.random() * contactsData.length)];
+        let include = contacts.includes(select);
+        const newContacts = [...contacts, select];
+        if(include == true){
+            addContact();
```

Comment:
Fizeram recursivo! bem legal, só vou levantar que recursivo precisamos ter mais uns cuidados ou ele vira um loop infinito e quebra a aplicação!

No caso aqui, isso acontece quando vcs tentam adicionar alguém depois que já foram todos adicionados!
Ou seja, o `include` sempre vai ser verdaeiro, que sempre vai re-chamar a função e vai ficar nisso infinitamente!

Dica: só um `if` resolve isso, no caso, "se" tal condição for verdade ele chama a função, caso contrario, ele não faz nada, assim quebrando o loop da recursividade.